flask-redis-centos7-atomicapp/artifacts/kubernetes/01-pod-redis.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: redis
+  labels:
+    name: redis
+    app: redis
+spec:
+  containers:
+  - name: redis
+    image: dharmit/redis
+    command:
+    - "redis-server"
+    ports:
+    - containerPort: 6379
+      hostPort: 6379
```

Comment:
In associated `docker` provider artifact, we did not use the host Port, do we need it? 
Even if we need it, do not hard code the `hostPort` value, always use the env vars.